src/Foundation/Maker.php

```diff
@@ -119,8 +119,9 @@ protected function getDependencies(array $parameters, array $primitives = [])
     {
         $dependencies = [];
 
+        /** @var ReflectionParameter $parameter */
         foreach ($parameters as $parameter) {
-            $dependency = $parameter->getClass();
+            $dependency = $parameter->getType();
```

Comment:
Reason for change?

Reply:
`getClass()` is deprecated, so checking against parameter type.